cogs/accounts.py

```diff
@@ -89,8 +89,7 @@ async def me(self,
         await interaction.response.send_message(embed=acc_details, delete_after=15, ephemeral=True)
 
     @slash_command(description="Register to Crypto Link", dm_permission=False)
-    @application_checks.check(has_wallet_inter_check())
-    @cooldowns.cooldown(1, 5, cooldowns.SlashBucket.author)
+    @cooldowns.cooldown(1, 5, cooldowns.SlashBucket.guild)
     async def register(self,
                        interaction: Interaction):
         if not self.backoffice.account_mng.check_user_existence(user_id=interaction.user.id):
```

Comment:
Missing message to user if IF check has failed. User needs to know why there was no response

Reply:
✔️